webpack/assets/javascripts/react_app/common/testHelpers.js

```diff
@@ -28,6 +28,20 @@ export default {
   },
 };
 
+export const mockWindowLocation = ({ href }) => {
```

Comment:
Does it still need after we added `visit` to `foreman_navigation`?
looks like the `visit` method get mocked now instead and it looks better.